aio/aio-proxy/aio_proxy/routes.py

```diff
@@ -36,9 +36,24 @@
 async def search_endpoint(request):
     try:
         terms, page, per_page, filters = extract_parameters(request)
-        total_results, unite_legale = search_es(
+        total_results, entreprises = search_es(
             Siren, terms, page * per_page, per_page, **filters
         )
```

Comment:
I am not sure : what is the best between hiding and selecting ?

I would put this logic in a separate wrapper function, called `hide_fields`or `select_fiels`, different from `extract_parameter`

Reply:
I think hiding is better, because it's actively removing a few fields rather than selecting a few.